pkg/ebpf/tracer.go

```diff
@@ -166,6 +169,14 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars exposes some metrics for tracer
+func (t *Tracer) setupExpvars() {
+	expvar.Publish("tracer.perf_received", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.perfReceived) }))
```

Comment:
`sysprobe.network.*` for all these as well

Reply:
Yea Sami just let me know 😭